src/visions/backends/spark/types/object.py

```diff
@@ -0,0 +1,11 @@
+from pyspark.sql.dataframe import DataFrame
+from pyspark.sql.types import DateType, StringType
+
+from visions.types.object import Object
+
+
+@Object.contains_op.register
+def object_contains(sequence: DataFrame, state: dict) -> bool:
+    assert len(sequence.schema) == 1
+    dtype = sequence.schema[0].dataType
+    return isinstance(dtype, (StringType, DateType))
```

Comment:
Should this include ComplexTypes (i.e. StructType, MapType, ArrayType) per https://spark.apache.org/docs/latest/sql-ref-datatypes.html